### DISCUSSION TITLE
License error: AuthorizationError: License expired

### DISCUSSION THREAD

**Discussion by jiepan1** [milaboratory/mixcr#1898] [General]
the  error: 
> mixcr --help
 License error: Authorization Error: License expired 
how can I inquire for offline license

**Comment by mizraelson** [comment]
Hi,

MiXCR requires internet access to periodically validate license key. For running MiXCR in the environment with restricted internet access, please add the following MiXCR IPs to the whitelist in the firewall settings:


IPv4:
75.2.96.100
99.83.215.63

IPv6:
2600:9000:a403:55e8:f9c1:f443:773b:4192
2600:9000:a51b:2483:639d:d5a9:270f:6439

**Comment by rashidma** [comment]
Hello,
I am using nextneopi and got this error:
Command error:
  IMPORTANT: MiXCR will use at most 12000MB of RAM,
             use -Xmx to change automatically set heap size (i.e. -Xmx50g to set heap size to 50 Gb)
  License manager thread died.
  License error: AuthorizationError: License expired

I was using ethernet with a firewall setting. Now, I am using a wifi. In both cases I am getting same error.
Please help.
Thanks

**Comment by mizraelson** [comment]
Hi, did you whitelist the IPs mentioned above?

```
Pv4:
75.2.96.100
99.83.215.63

IPv6:
2600:9000:a403:55e8:f9c1:f443:773b:4192
2600:9000:a51b:2483:639d:d5a9:270f:6439
```

**Reply by rashidma** [reply]
Hello mizraelson,
I did added the IPs using iptables. Please see below the output of "iptables =L":

Chain INPUT (policy ACCEPT)
target     prot opt source               destination         
ACCEPT     all  --  a3856255e003c403d.awsglobalaccelerator.com  anywhere            
ACCEPT     all  --  a3856255e003c403d.awsglobalaccelerator.com  anywhere            

Still mixcr is complaining as below:

Command error:
  IMPORTANT: MiXCR will use at most 12000MB of RAM,
             use -Xmx to change automatically set heap size (i.e. -Xmx50g to set heap size to 50 Gb)
  License manager thread died.
  License error: AuthorizationError: License expired


Please let me know how to solve this issue.

Thanks
